G/GLFW/build_tarballs.jl

```diff
@@ -5,22 +5,23 @@ version = v"3.3.2"
 
 # Collection of sources required to build glfw
 sources = [
-    "https://github.com/glfw/glfw/releases/download/$(version)/glfw-$(version).zip" =>
-    "08a33a512f29d7dbf78eab39bd7858576adcc95228c9efe8e4bc5f0f3261efc7",
+    ArchiveSource("https://github.com/glfw/glfw/releases/download/$(version)/glfw-$(version).zip",
+    "08a33a512f29d7dbf78eab39bd7858576adcc95228c9efe8e4bc5f0f3261efc7")
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
 cd $WORKSPACE/srcdir/glfw-*/
 mkdir build && cd build
-cmake .. -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE="${CMAKE_TARGET_TOOLCHAIN}" \
-    -DBUILD_SHARED_LIBS=ON \
-    -DCMAKE_BUILD_TYPE=Release \
-    -DGLFW_BUILD_EXAMPLES=false \
-    -DGLFW_BUILD_TESTS=false \
-    -DGLFW_BUILD_DOCS=OFF
+CMAKE_FLAGS="-DCMAKE_INSTALL_PREFIX=${prefix} -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release"
+CMAKE_FLAGS="${CMAKE_FLAGS} -DBUILD_SHARED_LIBS=ON"
+CMAKE_FLAGS="${CMAKE_FLAGS} -DGLFW_BUILD_EXAMPLES=false"
+CMAKE_FLAGS="${CMAKE_FLAGS} -DGLFW_BUILD_TESTS=false"
+CMAKE_FLAGS="${CMAKE_FLAGS} -DGLFW_BUILD_DOCS=OFF"
+cmake ${CMAKE_FLAGS} ..
 make -j${nproc}
 make install
+install_license ../LICENSE.md
```

Comment:
For what is worth, I find the previous script more readable and the license file should be installed automatically, so I don't really see the need of this block of change.

Reply:
Nothing special. I just copied the style used in the old [LLVMBuilder](https://github.com/staticfloat/LLVMBuilder/blob/master/build_tarballs.jl#L180). Feel free to revert. :)

Reply:
> I just copied the style used in the old LLVMBuilder

Which is massively more complicated, needs lots of conditionals, probably needs to manually install the license file.  I wouldn't look at that builder as a good example, just because it's very sophisticated and has special needs.